Makefile.dep

```diff
@@ -325,3 +325,8 @@ ifneq (,$(filter cpp11-compat,$(USEMODULE)))
   USEMODULE += timex
   FEATURES_REQUIRED += cpp
 endif
+
+ifneq (,$(filter hih6130,$(USEMODULE)))
+  USEMODULE += vtimer
+  FEATURES_REQUIRED += periph_i2c
```

Comment:
Ahh, I forgot this one, sorry. With #3420 it is surely the correct place for the dependencies. I tend to take the `FEATURES_REQUIRED` out here. I think we used to put it in an applications makefile to throw a warning if features don't match (as you already did).